.travis.yml

```diff
@@ -10,11 +10,29 @@ os:
 
 install:
 - if [[ "$TRAVIS_OS_NAME" == "osx" ]]; then brew install make; fi
-- if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then sudo apt-get install make; fi
+- if [[ "$TRAVIS_OS_NAME" == "linux" ]]; then
+    sudo apt-get install make;
```

Comment:
I think make is already present?

Reply:
maybe - I assumed not since this was part of the code